rpc/client_test.go

```diff
@@ -84,8 +84,8 @@ func TestClientErrorData(t *testing.T) {
 	// Check code.
 	if e, ok := err.(Error); !ok {
 		t.Fatalf("client did not return rpc.Error, got %#v", e)
-	} else if e.ErrorCode() != (testError{}.ErrorCode()) {
-		t.Fatalf("wrong error code %d, want %d", e.ErrorCode(), testError{}.ErrorCode())
+	} else if e.ErrorCode() != internalServerErrorCode {
+		t.Fatalf("wrong error code %d, want %d", e.ErrorCode(), internalServerErrorCode)
 	}
 	// Check data.
 	if e, ok := err.(DataError); !ok {
```

Comment:
Not sure if we intend to return internal error data from services. Didn't find any of our service implementation implement ErrorData.

Reply:
I'm not sure what you are asking here. This test just verifies that the error `"data"` field is returned at all when the server sends it.

Reply:
My question was whether this is by design to return the error from the internal service directly. I didn't find existing services return `rpc.Error`. Thanks for confirming :-)